src/quantum/widget.jsx

```diff
@@ -4,102 +4,115 @@ import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-const enrich = (text, key = 'none') => (typeof text === 'string'
-  ? (
+const enrich = (text, key = 'none') =>
+  typeof text === 'string' ? (
     <span
       key={`enriched-${key}`}
+      // eslint-disable-next-line react/no-danger
       dangerouslySetInnerHTML={{
           // eslint-disable-line
         __html: text
-            .replace(/\*([^*]+)\*/g, '<em>$1</em>')
-            .replace(
-              /\[([^\]]+)\]\(([^)]+)/g,
-              '<a href="$2" target="_blank" rel="noopener noreferrer">$1</em>',
-            ),
+          .replace(/\*([^*]+)\*/g, '<em>$1</em>')
+          .replace(
+            /\[([^\]]+)\]\(([^)]+)/g,
+            '<a href="$2" target="_blank" rel="noopener noreferrer">$1</em>'
+          ),
       }}
     />
-)
-  : text);
+  ) : (
+    text
+  );
 
 export default class Widget extends React.Component {
   render() {
     const title = enrich(this.props.title);
     // const updated = this.props.updated && moment(this.props.updated);
-    const $title = this.props.link
-      ? (
-        <a href={this.props.link} target='_blank' rel='noopener noreferrer'>
-          {title}
-        </a>
-)
-      : title;
+    const $title = this.props.link ? (
```

Comment:
While you're here, can you remove the commented out code on line 29?

Reply:
done.